backend/src/externalAPI/externalAPI.service.ts

```diff
@@ -78,21 +77,23 @@ export class ExternalAPIService {
     userID: string,
   ): Promise<string> {
     try {
-      this.checkAllowed(userID);
+      await this.checkAllowed(userID);
       const paramsCheck = await this.checkParams(prompt, groupID);
       if (paramsCheck) {
+        console.log('REQUEST WOLFRAM', prompt);
```

Comment:
Consider using NestJS's logger instead the default console.log() statement.

The logger can be used like this
```typescript
 Logger.log('message', 'context');
```
The context in this case would be just "ExternalAPI" and the message "REQUEST WOLFRAM"

Reply:
Fixed in commit e075875